README.md

```diff
@@ -150,13 +163,15 @@ def stats(message, start_date=None, end_date=None):
 ```
 
 
-And add the plugins module to `PLUGINS` list of slackbot settings, e.g. slackbot_settings.py:
+And add the relative path to the plugins modules you created at the slackbot instantiation:
```

Comment:
`relative path` is inaccurate here. Each element of the `plugins` list is a string representing a python module.